testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -3267,29 +3267,29 @@ QUnit.module('default options', {
 
             $(lookup.field()).trigger('dxclick');
 
-            assert.roughEqual($popup.find('.dx-overlay-content').position().top, -2.5, 1, 'offset of the lookup if last item is selected');
+            assert.roughEqual($popup.find('.dx-overlay-content').position().top, -2.5, 1.001, 'offset of the lookup if last item is selected');
 
             lookup.close();
 
             lookup.option('dataSource', []);
 
             $(lookup.field()).trigger('dxclick');
 
-            assert.roughEqual($popup.find('.dx-overlay-content').position().top, 0, 1, 'offset of the lookup if not selected item');
+            assert.roughEqual($popup.find('.dx-overlay-content').position().top, 0, 1.001, 'offset of the lookup if not selected item');
```

Comment:
`1 -> 1.001`
Do we need to change these tests?
They pass locally without changing this value

Reply:
Unfortunately, it looks like a flasher. There are some problems with this test on drone

Reply:
Why we do not see these test fails on the 20_1 branch?

Reply:
I brought them back. Hope, it was only strange coincidence, when tests failed